### PR TITLE
Update Bluesocket to latest (1.0.46)

### PR DESCRIPTION
**Pull request by jnewc**
Bluesocket was causing version conflicts in one of my projects so I've upgraded to the latest.

### CODE REVIEW

**Comment by beefon** [comment]
Prepared my PR as well (#26) and didn't notice yours. :)

File: Package.swift [hunk 1]
```diff
@@ -9,7 +9,7 @@ let package = Package(
         .library(name: "Shout", targets: ["Shout"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/IBM-Swift/BlueSocket", from: "0.12.68"),
+        .package(url: "https://github.com/IBM-Swift/BlueSocket", from: "1.0.0"),
```
Comment (beefon) [inline Package.swift]: Why from 1.0.0? I'd set it to be from 1.0.46, the most recent version available.

**Review by beefon — APPROVED** [review]
Wish to merge this asap, want to get rid from my fork.

**Comment by CognitiveDisson** [comment]
Pls merge this PR, it's block my work.

**Comment by ignatovv** [comment]
+1 for this pull request

File: Package.swift (hunk 1 above)
Comment (artyom-razinov) [reply Package.swift]: +1

**Comment by artyom-razinov** [comment]
+1

**Review by artyom-razinov — APPROVED** [review]

File: Package.swift [hunk 2]
```diff
@@ -6,10 +6,10 @@ import PackageDescription
 let package = Package(
     name: "Shout",
     products: [
-        .library(name: "Shout", targets: ["Shout"]),
+        .library(name: "Shout", type: .static, targets: ["Shout"]),
```
Comment (jakeheis) [inline Package.swift]: What's the rationale for this change?

Comment (beefon) [reply Package.swift]: Oh, that’s surprising, this has been pushed after my review. I’m strongly against this change though.

**Review by beefon — CHANGES_REQUESTED** [review]
Please remove `.static` as this is not appropriate for this PR. Library is linked statically by default anyways.

File: Package.swift (hunk 2 above)
Comment (jnewc) [reply Package.swift]: Sorry, change I was trying on my fork that crept into the PR. I'll revert.

**Comment by jnewc** [comment]
@beefon @jakeheis the requested changes have been made.

**Comment by jakeheis** [comment]
Thanks!
